frontend/src/components/details/DetailsComponent.js

```diff
@@ -45,6 +46,9 @@ class DetailsComponent extends Component {
             modified: 'modified Scenario'
         };
         const { selectedProduct } = this.props;
+
+        postCalculationRequest(selectedProduct.productID);
+
```

Comment:
This will need to be moved to the correct place, once the getters for the Graphs are implemented.

@ScoutAtlas We will need the productId to execute the calculation stuff, and we should probably not use the one stored in selectedProduct, as that might change, and we will have to handle multiple Products side by side on the DetailsPage.

Reply:
@Waldleufer what exactly needs to be calculated? The change via variables (modified scenarios)?

But aren't we always comparing a single product and a single model? Only different scenarios of a model.